variables.tf

```diff
@@ -1,49 +1,23 @@
-variable "aws_region" {
-  description = "AWS region."
-  type        = "string"
-}
-
-variable "environment" {
-  description = "A name that identifies the environment, will used as prefix and for tagging."
-  type        = "string"
-}
-
-variable "vpc_id" {
-  description = "The VPC that is used for the instances."
-  type        = "string"
-}
-
-variable "subnet_id_runners" {
-  description = "Subnet used to hosts the docker-machine runners."
-  type        = "string"
-}
-
 variable "availability_zone_runners" {
   description = "Availability zone used to host the docker-machine runners."
   type        = "string"
   default     = "a"
 }
 
-variable "availability_zone_proxy" {
-  description = "Availability zone used to host the proxy."
-  type        = "string"
-  default     = "a"
-}
-
-variable "subnet_id_gitlab_runner" {
-  description = "Subnet used for hosting the gitlab-runner."
+variable "aws_region" {
```

Comment:
why this module contains variable to configure AWS Region?

Reply:
Refactored: getting current aws_regoin from data now